securedrop/journalist_app/forms.py

```diff
@@ -2,8 +2,44 @@
 
 from flask_babel import gettext
 from flask_wtf import FlaskForm
-from wtforms import TextAreaField
-from wtforms.validators import InputRequired
+from wtforms import (TextAreaField, TextField, BooleanField, HiddenField,
+                     ValidationError)
+from wtforms.validators import InputRequired, Optional
+
+from db import Journalist
+
+
+def otp_secret_validation(form, field):
```

Comment:
We could regex this to ensure it's hex encoded to catch errors before they hit the app code. This isn't a blocker, but it would be nice because it would put all the errors in the same place.

Reply:
Good idea - want to file a followup ticket for that?